docs/rfcs/0000-rfc-template.md

```diff
@@ -0,0 +1,126 @@
+# RFC-NNNN: Human-readable title goes here
+
+Remember to add autogenerated HackMD badge here. See [README.md](README.md) for more details.
+
+**Authors** (in alphabetical order):
+
+- Author Name, [@Author_GitHub_Handle](https://github.com/Author_GitHub_Handle)
+
+**Status** (as defined [here]): `Provisional`
+
+[here]: https://github.com/kubernetes/enhancements/blob/master/keps/0001-kubernetes-enhancement-proposal-process.md#kep-metadata
+
+**Creation Date**: `YYYY-MM-DD`
+
+**Last Updated**: `YYYY-MM-DD`
+
+**RFC Handle**: `rfc-template` (should match the file name, as `NNNN-{rfc_handle}`)
+
+**Initial Pull Request**: [racklet/racklet#NNNN](https://github.com/racklet/racklet/pull/NNNN)
+
+**Tracking Issue**: [racklet/racklet#NNNN](https://github.com/racklet/racklet/issues/NNNN)
+
+[TOC]
+
+## Summary
+
+One paragraph explanation of the feature.
+
+## Motivation
+
+Why are we doing this? What use cases does it support? What is the expected outcome?
+
+### Goals
+
+### Non-Goals
+
+## Proposal
+
+This is the technical portion of the RFC. Explain the design in sufficient detail that:
+
+- Its interaction with other features is clear.
+- It is reasonably clear how the feature would be implemented.
+- Corner cases are dissected by example.
+
+The section should return to the examples given in the guide level explanation below, and explain more fully how the detailed proposal makes those examples work.
+
+### Values
+
+Describe what values does the proposed feature reflect. See RFC-0001[FIXME: Add link to RFC1].
+
+### User stories
+
+Explain what is the use case of the proposed feature and how it would benefit the user.
+
+### Guide-level explanation
+
+Explain the proposal as if it was already a feature of the project and this would be the documentation for that feature.
+
+- Introducing new named concepts.
+- Explaining the feature largely in terms of examples.
+- If applicable, provide sample error messages, deprecation warnings, or migration guidance.
+- If applicable, describe the differences between teaching this to a Racklet administrator versus a Racklet end user.
+
+### Risks and Mitigations
+
+What are the risks of this proposal and how do we mitigate.
+Think broadly.
+For example, consider both security and how this will impact the larger kubernetes ecosystem.
```

Comment:
Just checking, is "kubernetes ecosystem" here intentional?

Reply:
No, good catch :+1: